src/ring/middleware/webjars.clj

```diff
@@ -1,20 +1,27 @@
 (ns ring.middleware.webjars
   (:import [org.webjars WebJarAssetLocator])
-  (:require [ring.middleware.head :as head]
+  (:require [clojure.string :as string]
```

Comment:
Usually `clojure.string` is abbreviated to `str`, though I think it's fine just to use `clojure.core/str` to join the file names.